test/subset-band-name/version.txt

```diff
@@ -0,0 +1 @@
+1.1.0
```

Comment:
Is this the right version?  or are you trying to keep it up to date with the service version?

Reply:
I guess it is. Roosevelt versioned it.

Reply:
Just to check in on this conversation - I also suspect that this is not the correct version number. At first I wondered if it had been made to match the version of the service itself, but that is only 0.6.0 in UAT.

Generally, the version number of a regression test image is decoupled from the version number of the service itself. I don't think this is a big, big deal, but I'd start this number either at 0.0.1 (like the other regression test images started) or maybe 1.0.0 (to indicate this is ready for prime time).

Reply:
I changed it to 1.1.0 because I inherited the test suite and made changes. I do not have a problem changing to 0.0.1.